peachjam/timeline_email_service.py

```diff
@@ -22,12 +24,20 @@
 class TimelineEmailService:
     @staticmethod
     def already_alerted_today(user, event_type):
+        q = Q()
         last_24_hrs = timezone.now() - timedelta(hours=24)
-        email_sent = TimelineEvent.objects.filter(
+
+        if type(event_type) == list:
+            q = Q(event_type__in=event_type)
+        elif type(event_type) == TimelineEvent.EventTypes:
```

Comment:
```suggestion
        else:
```

Otherwise, if you have a typo somewhere and you're calling this method with a weird value for `event_type`, then you'll never apply an `event_type` filter to `q` which would be dangerous.